docs/3.0.0-beta.x/guides/error-handling.md

```diff
@@ -0,0 +1,89 @@
+# Error handling
+
+In this guide we will see how you can handle errors to send it on the Application Monitoring / Error Tracking Software you want.
+
+::: note
+In this example we will use [Sentry](https://sentry.io).
+:::
+
+## Create a middleware
+
+To handle errors, we will have to use a [middleware](../concepts/middlewares.md) that will catch errors and send them to Sentry.
```

Comment:
A middleware will be used in order to catch the errors which will then be sent to Sentry.